nodejs/nodejs-cloud-run-hello-world/CONTRIBUTING.md

```diff
@@ -0,0 +1,25 @@
+# Contributing
+
+Join us at https://github.com/GoogleCloudPlatform/cloud-code-samples.
+
+## Run the Tests
+
+The tests for this code are implemented as a [Cloud Build](https://cloud.google.com/cloud-build) pipeline which takes the following steps:
+
+* Build the service
+* Deploy to Cloud Run
+* Run the tests
+* Delete the service and container image
+
+To run this end-to-end test manually, run this command-line operation:
+
+```sh
+gcloud builds submit . --config cloudbuild.yaml --substitutions COMMIT_SHA=manual
+```
+
+You can run the tests against a locally running instance of the service:
+
+```sh
+GOOGLE_CLOUD_PROJECT=local npm start
+npm test
```

Comment:
I don't think we need this file here. We can stick to having a single CONTRIBUTING.md at the root of this repo and include this information there across all the languages. Otherwise we'll have users clone the template and have this file being part of the template of a new app.

Reply:
Discussed in chat to create one per language.